internal/pikamon/commands/catch.go

```diff
@@ -2,64 +2,65 @@ package commands
 
 import (
 	"fmt"
-	"github.com/Jac0bDeal/pikamon/internal/pikamon/util"
-	"github.com/bwmarrin/discordgo"
-	"github.com/mtslzr/pokeapi-go"
-	log "github.com/sirupsen/logrus"
 	"math/rand"
 	"strconv"
 	"strings"
+
+	"github.com/Jac0bDeal/pikamon/internal/pikamon/constants"
+	"github.com/bwmarrin/discordgo"
+	"github.com/mtslzr/pokeapi-go"
+	log "github.com/sirupsen/logrus"
 )
 
 var pokemonExpiredMessages = []string{
 	"The pokemon heard you coming and ran for the hills!",
-	"The pokemon thought you were ugly and didn't want to be caught by an ugly person!",
-	"The pokemon likes Physics (mistakenly) and thought you were too dumb to understand it so it ran!",
+	"The pokemon got scared and fled!",
```

Comment:
not sure how I feel about this change 🤣

Reply:
well, people weren't liking being insulted by a bot haha